assets/grafana/celeborn-dashboard.json

```diff
@@ -11368,6 +11368,98 @@
           "title": "metrics_replicate_pinnedDirectMemory_Value",
           "type": "timeseries"
         },
+        {
```

Comment:
The panel of `CacheHitRate` should be put after the panels of sort metrics.

Reply:
PTAL.